src/main/java/com/spring/exercise/utils/CustomValidators/Age.java

```diff
@@ -0,0 +1,25 @@
+package com.spring.exercise.utils.CustomValidators;
+
+import com.spring.exercise.utils.AppMessages;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target({ FIELD })
+@Retention(RUNTIME)
+@Constraint(validatedBy = AgeValidator.class)
+@Documented
+public @interface Age {
```

Comment:
tak sobie myślę, czy nie lepiej by było sparametryzować tej adnotacji, by dało się dać min i max age value. Dzięki temu nie byłoby to takie sztywne